### ISSUE TITLE
Windfarm wordwide...

### ISSUE THREAD

**Issue by Geoneer** [Ysurac/FlightAirMap#376]
https://earth.nullschool.net/ - check! Great for your airplane routes to show on the map..

![image](https://user-images.githubusercontent.com/218929/33563930-81fd8be6-d919-11e7-8a85-3a56851e1f34.png)

**Comment by Ysurac** [comment]
Some code/idea used for 2D winds display come from this project in fact. 
But this will be more difficult to use that in CesiumJS.

**Comment by Geoneer** [comment]
yes I know. I did contact #devteam @cesiumjs to collaborate in this request. So maybe we can share some effort to create something like this.
